interfaces/Matlab_Toolbox_Revamp/Examples/conhp.m

```diff
@@ -0,0 +1,27 @@
+function dydt = conhp(t, y, gas, mw) %#ok<INUSL>
```

Comment:
Can you integrate this function within the `ignite_hp.m` example file? Likewise for `conuv.m`.

Reply:
Done!